common/arg.cpp

```diff
@@ -966,6 +966,27 @@ gpt_params_context gpt_params_parser_init(gpt_params & params, llama_example ex,
             params.sparams.min_p = std::stof(value);
         }
     ).set_sparam());
+    add_opt(llama_arg(
+        {"--xtc-p"}, "N",
+        format("xtc probability (default: %.1f, 0.0 = disabled)", (double)params.sparams.xtc_p),
+        [](gpt_params & params, const std::string & value) {
+            params.sparams.xtc_p = std::stof(value);
+        }
+    ).set_sparam());
+    add_opt(llama_arg(
+        {"--xtc-t"}, "N",
+        format("xtc threshold (default: %.1f, 0.0 = disabled)", (double)params.sparams.xtc_t),
```

Comment:
It technically is correct, and both ways, I will fix it both in description and in the sampler itself. It doesn't make sense both as 0 (all tokens affected) and 1 (no tokens affected).

Reply:
Setting `threshold` to 0 should not disable XTC. It should eliminate all tokens except for the least probable one, in line with the XTC mechanism, since all tokens have probability >= 0. Anything else is an unnecessary and confusing special case, given that setting `probability` to 0 already disables XTC in a natural and intuitive manner.

And there are cases where it *does* make sense to do this. In fact, it's quite an interesting option to have when combining XTC with aggressive truncation.

Reply:
Yes, sorry, I've missed this comment somehow, will fix.